VideoLocker/src/androidTest/java/org/edx/mobile/test/feature/interactor/RegistrationScreenInteractor.java

```diff
@@ -0,0 +1,98 @@
+package org.edx.mobile.test.feature.interactor;
+
+import android.support.test.espresso.ViewInteraction;
+
+import org.edx.mobile.R;
+import org.edx.mobile.base.MainApplication;
+import org.edx.mobile.core.EdxEnvironment;
+import org.edx.mobile.module.registration.model.RegistrationDescription;
+import org.edx.mobile.module.registration.model.RegistrationFormField;
+import org.edx.mobile.test.feature.custom.SpinnerViewAction;
+import org.edx.mobile.test.feature.data.Credentials;
+import org.edx.mobile.util.Config;
+import org.edx.mobile.util.ResourceUtil;
+
+import static android.support.test.espresso.Espresso.onView;
+import static android.support.test.espresso.action.ViewActions.click;
+import static android.support.test.espresso.action.ViewActions.typeText;
+import static android.support.test.espresso.assertion.ViewAssertions.matches;
+import static android.support.test.espresso.matcher.ViewMatchers.isCompletelyDisplayed;
+import static android.support.test.espresso.matcher.ViewMatchers.withHint;
+import static android.support.test.espresso.matcher.ViewMatchers.withId;
+import static android.support.test.espresso.matcher.ViewMatchers.withText;
+
+public class RegistrationScreenInteractor {
+
+    RegistrationDescription registrationFields;
+
+    public RegistrationScreenInteractor observeRegistrationScreen() {
+        final MainApplication app = MainApplication.instance();
+        try {
+            registrationFields = app.getInjector().getInstance(EdxEnvironment.class).getServiceManager().getRegistrationDescription();
+        } catch (Exception e) {
+            e.printStackTrace();
+        }
+        final CharSequence title = ResourceUtil.getFormattedString(app.getResources(), R.string.register_title, "platform_name", app.getInjector().getInstance(Config.class).getPlatformName());
+        onView(withId(R.id.activity_title)).check(matches(withText(title.toString())));
+        onEmailView().check(matches(isCompletelyDisplayed()));
```

Comment:
This can be revisited after implementing Akiva's suggestion to add an identifier field here: https://github.com/edx/edx-app-android/pull/673#discussion_r61167004